Tests/PublishTests/Tests/SiteMapGenerationTests.swift

```diff
@@ -57,6 +60,8 @@ final class SiteMapGenerationTests: PublishTestCase {
 
         let unexpectedLocations = [
             "https://swiftbysundell.com/one/itemB",
+            "https://swiftbysundell.com/two/itemC",
```

Comment:
I'd also add the section's index location here as well:
```
https://swiftbysundell.com/two
```
Just to protect against future regressions. I think it's also worth documenting that adding a section's path to the list of excluded paths removes that entire section (including its items).

Reply:
Sounds good 👍